container.go

```diff
@@ -420,12 +420,14 @@ func createContainer(pod *Pod, contConfig ContainerConfig) (*Container, error) {
 		return nil, err
 	}
 
-	agentCaps := c.pod.agent.capabilities()
-	hypervisorCaps := c.pod.hypervisor.capabilities()
+	if !c.pod.config.HypervisorConfig.DisableBlockDeviceUse {
```

Comment:
Please rebase, as this has been merged in a separate PR.

Reply:
Rebased. Moved some code from qemu.go to qemu_arch_base.go introduced with arm support.